src/config/templates.toml

```diff
@@ -8,7 +8,7 @@ separate(" ",
 
 log = '''
 label(if(current_working_copy, "working_copy"),
-  concat(
+  separate("\n",
```

Comment:
Since the second line still requires the `++ "\n"`, I think this looks less consistent.

Also, since `separate()` drops empty elements, the reader now has to think whether the entire line can be empty to understand what's going to be rendered. I don't think it's going to be different in practice since we always set a description for the operation, so my concern is just that it forces the reader think about it.